staff/sean-daryanani/cv.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Sean Curriculum</title>
+</head>
+
+<body>
+    <header>
+        <h1>Sean Daryanani | Aspiring Web Developer</h1>
+        <address>
+            <p><strong>Calle</strong>: XXXXXXXXXXX</p>
+            <p><strong>Ciudad</strong>:Barcelona</p>
+            <p><strong>Email</strong>: <a href="mailto:seand52@gmail.com?Subject=Job Opportunity" target="_top">seand52@gmail.com</a></p>
+            
+        </address>
+        <hr>
+    </header>
+
+    <section>
+        <h2>Career Goals</h2>
+        <p>I am an aspiring developer currently undertaking a web development bootcamp at Skylab Academy. I believe my
+            hard-working attitude will allow me to thrive in this field.</p>
+    </section>
+
+    <section>
+        <h2>Skills</h2>
+        <ol>
+            <li>Programming Languages
+                <ul>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                    <li>Python</li>
+                    <li>R</li>
+                </ul>
+            </li>
+            <li>Data analysis Software
+                <ul>
+                    <li>Excel</li>
+                    <li>SPSS</li>
+                    <li>Q Research</li>
+                </ul>
+            </li>
+        </ol>
+    </section>
+
+    <section>
+        <h2>Projects</h2>
+        <ul>
+            <li>Pasapalabra Game - This game is was made using HTML, CSS, and Javascript</li>
+            <li>Calculator - Web-based calculator that uses HTML, CSS, and Javascript</li>
+        </ul>
+    </section>
+
+    <section>
+        <h2>Work Experience</h2>
+        <table>
```

Comment:
you seem to be using `table` element as a layout solution, but not as the correct purpose of this element, which is just for listing data in table format. please, if that's the case, correct it. and RTFM https://www.w3schools.com/html/html_tables.asp